README.md

```diff
@@ -109,6 +109,15 @@ a short grace period during a libdef installation, but sometimes it is useful to
 do this update manually. Use this command if you want to download the most
 recent definitions into the cache for yourself.
 
+##### `flow-typed create-stub package-specification [--typescript]`
+
+Creates `any`-typed libdef that you can fill in.
+
+If the `--typescript` flag is specified, the libdef would be converted from `typings` field in package.json, `index.d.ts` file or `@types/package-name` package to flow-typed libdef format.
+
+Note that this option can produce unpredictable output because TypeScript-to-Flow converter is not perfect.
```

Comment:
Maybe also mention that manual fixes and careful reviewing of the output is needed, and potentially suggestion about contributing them back to flow-typed?

Reply:
Isn't this already adressed with `can produce unpredictable output`

Reply:
I think it can't be emphasised enough.

Reply:
I updated it